ROSCO_toolbox/controller.py

```diff
@@ -515,9 +515,9 @@ def peak_shaving(self,controller, turbine):
     def min_pitch_saturation(self, controller, turbine):
         
         # Find TSR associated with minimum rotor speed
-        TSR_at_minspeed = (controller.pc_minspd/turbine.Ng) * turbine.rotor_radius / controller.v_below_rated
```

Comment:
@nikhar-abbas, I think we want to revert this to match `develop` but I don't remember the history here.  What should it be?

Reply:
I don't quite remember the history here either. We're defining `controller.pc_minspd = controller.vs_minspd` [here](https://github.com/NREL/ROSCO_toolbox/blob/2cc92c9a4c11545df023cde81771787d0e962375/ROSCO_toolbox/controller.py#L284), though this probably redundant and unnecessary. 

So, I like the change you made here - maybe just delete the line linked above and it should be good.